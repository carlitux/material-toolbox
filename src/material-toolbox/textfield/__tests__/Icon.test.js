// @flow
import React from 'react';
import { shallow } from 'enzyme';
import TextFieldIcon from '../Icon';

describe('component::TextFieldIcon', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<TextFieldIcon />);
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot cssOnly', () => {
    const component = shallow(<TextFieldIcon cssOnly />);
    expect(component).toMatchSnapshot();
  });
});
