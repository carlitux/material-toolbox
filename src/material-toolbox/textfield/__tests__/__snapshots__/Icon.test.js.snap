// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::TextFieldIcon Should match snapshoot 1`] = `
<i
  className="mdc-text-field__icon"
  onClick={[Function]}
  onKeyDown={[Function]}
  tabIndex="-1"
/>
`;

exports[`component::TextFieldIcon Should match snapshoot cssOnly 1`] = `
<i
  className="mdc-text-field__icon"
  onClick={[Function]}
  onKeyDown={[Function]}
  tabIndex="-1"
/>
`;
